src/client/__tests__/services/gitHubService.test.ts

```diff
@@ -0,0 +1,37 @@
+import {anyString, instance, mock, when} from 'ts-mockito';
+import GitHub from 'github-api';
+import {GitHubService, IGithubService} from '../../services/gitHubService';
+import {SocialStore} from '../../store/SocialStore';
+import {buildGetRepositoryResponse} from '../testKits/apis/githubApi';
+
+describe('Social Data in the app', () => {
```

Comment:
description doesn't really describe this test

Reply:
Fix